src/main/java/com/google/codeu/servlets/StatsPageServlet.java

```diff
@@ -38,4 +40,62 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     jsonObject.addProperty("messageCount", messageCount);
     response.getOutputStream().println(jsonObject.toString());
   }
-}
\ No newline at end of file
+  
+  /*
+   * Obtaining the average message by going through all the list
+   */
+  public void avgMessage(HttpServletRequest request, HttpServletResponse response) 
+		  throws IOException{
```

Comment:
nit: Please reformat this code to use spaces instead of tabs like the rest of the code and make it match indentation of the other code. The key think here is that it shouldn't be possible to notice the difference between code you added and the other developers. It also looks like you should add spaces in various if & for statements. The original code primarily follows Google Java style guide (https://google.github.io/styleguide/javaguide.html)